.github/workflows/libretro.yml

```diff
@@ -0,0 +1,35 @@
+name: Libretro cores
+
+on:
+  push:
+    paths: 'src/libretro/**'
+  pull_request:
+    paths: 'src/libretro/**'
+
+jobs:
+  libretro:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: build linux & win32 cores
+        run: |
+          sudo apt-get update && sudo apt-get install -y mingw-w64
+          cd src/libretro && make
+          mv freej2me_libretro.so freej2me_libretro_linux.so
+          make clean
+          platform=win32 CC=x86_64-w64-mingw32-gcc CPP=x86_64-w64-mingw32-g++ CXX=x86_64-w64-mingw32-g++ make
+          mv freej2me_libretro_linux.so freej2me_libretro.so
```

Comment:
Minor thing here, but is this needed because of Actions' environment? Usually the win32 build command should result in a dll directly, which means that renaming .so files temporarily shouldn't be needed...

That's just because i noticed that this file gets effectively renamed to "freej2me_libretro_linux.so" and then back to "freej2me_libretro.so", but don't know why this is being done here... feedback would be appreciated!

Reply:
> This CI builds both Linux .so and Win32 .dll, not just dll. The reason there are two mvs is that the build env will need to clean up the .o files for the last build for Linux so, so that the next run of building dll can proceed successfully, otherwise dll build will fail since it will attempt to re-use the .o file already there.
> 
> However, if one runs `make clean` without renaming the so file, the so file will also get removed, so this is just to preserve it.

Oh, i see... wouldn't a "rm" command to remove that intermediate object file be enough though? Your solution already works fine don't get me wrong, i'm just looking for an alternative that's easier to understand, even if that's not really needed here.

> An alternative way is to have 2 build jobs and run the Win32 build in MSYS2 separately, but this will take longer time for CI to setup. If you do want that way instead, let me know.

Nah, not really, just wanted to understand the reasoning behind that temporary rename, that's all. The quicker this runs, the better, especially for when i'm on a commit frenzy.